frontend/src/lib/constants.tsx

```diff
@@ -91,19 +91,17 @@ export const WEBHOOK_SERVICES: Record<string, string> = {
 export const FEATURE_FLAGS = {
     // Cloud-only
     CLOUD_ANNOUNCEMENT: 'cloud-announcement',
-    NPS_PROMPT: '4562-nps', // owner: @paolodamico
+    NPS_PROMPT: '4562-nps', // owner: @marcushyett-ph
     // Experiments / beta features
-    INGESTION_GRID: 'ingestion-grid-exp-3', // owner: @kpthatsme
+    INGESTION_GRID: 'ingestion-grid-exp-3', // owner: @liyiy
```

Comment:
FYI @liyiy as you now own this piece, I think it's best if you decide how to move forward with this.

Reply:
I think that stuff looks redundant with our current PreflightCheck? It looks like it should just be removed along with the `IngestionWizard` code